### PR TITLE
Updated with BCParks.ca websites & pages content types as well as initial data load. Also included is CKEditor build 

### PR DESCRIPTION
**Pull request by nileshnad**
# Description

This PR includes the following proposed change(s):

- BC-258 Sample home page
- BC-315 Strapi HTML Editor

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by arbalisi12 — CHANGES_REQUESTED** [review]
Hi Nilesh,

Please resolve the conflicts before merging.

Thanks,
Arnel

**Review by arbalisi12 — APPROVED** [review]
looks good. thanks

File: src/cms/api/page/config/routes.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+<<<<<<< HEAD:src/cms/api/page/config/routes.json
+      "path": "/pages",
+      "handler": "page.create",
+=======
+      "path": "/park-photos",
+      "handler": "park-photo.create",
+>>>>>>> ef7a23f543c40efd414fd7448de3fc9c171e6a55:src/cms/api/park-photo/config/routes.json
+      "config": {
+        "policies": []
+      }
```
Comment (siva-ka) [inline src/cms/api/page/config/routes.json:33]: Issues with merge

**Review by siva-ka — APPROVED** [review]
sorry, was looking at old version. All good, thanks Nilesh :)

**Review by nileshnad — COMMENTED** [review]
yes the routes json files between park photos, park names, website and page content types were all messed up. I have committed the changes

**Pull request merged**